components/servo/lib.rs

```diff
@@ -773,11 +803,21 @@ where
                 }
             },
 
-            EmbedderEvent::WebViewVisibilityChanged(webview_id, visible) => {
-                let msg = ConstellationMsg::WebViewVisibilityChanged(webview_id, visible);
+            EmbedderEvent::MarkWebViewInvisible(webview_id) => {
+                let msg = ConstellationMsg::MarkWebViewInvisible(webview_id);
+                if let Err(e) = self.constellation_chan.send(msg) {
+                    warn!(
+                        "Sending MarkWebViewInvisible to constellation failed ({:?}).",
+                        e
+                    );
+                }
+            },
+
+            EmbedderEvent::UnmarkWebViewInvisible(webview_id) => {
+                let msg = ConstellationMsg::UnmarkWebViewInvisible(webview_id);
                 if let Err(e) = self.constellation_chan.send(msg) {
                     warn!(
-                        "Sending WebViewVisibilityChanged to constellation failed ({:?}).",
+                        "Sending UnmarkWebViewInvisible to constellation failed ({:?}).",
```

Comment:
Sorry this suggestion won't apply cleanly due to a limitation of GitHub:

```suggestion
                   warn!("Sending UnmarkWebViewInvisible to constellation failed ({error:?}).");
```

Reply:
Fixed in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)